src/catalog/abstract_catalog.cpp

```diff
@@ -104,6 +104,10 @@ bool AbstractCatalog::InsertTuple(std::unique_ptr<storage::Tuple> tuple,
 bool AbstractCatalog::DeleteWithIndexScan(oid_t index_offset,
                                           std::vector<type::Value> values,
                                           concurrency::Transaction *txn) {
+  LOG_DEBUG("Delete following values in index %d", index_offset);
```

Comment:
Can I simply remove these debug info? I don't see LOG_DEBUG used so frequently outside of triggers.